include/deal.II/fe/fe_tools.h

```diff
@@ -841,148 +841,234 @@ namespace FETools
 
 
   /**
-   * Take vectors of finite elements and multiplicities and multiply out
-   * how many degrees of freedom the composed element has per vertex,
-   * line, etc.
-   *
-   * If @p do_tensor_product is true, the number of components
-   * returned in the FiniteElementData object is the sum over the
-   * product of the number of components in each of the finite
-   * elements times the corresponding multiplicity.  Otherwise the
-   * number of components is taken from the first finite element with
-   * non-zero multiplicity, and all other elements with non-zero
-   * multiplicities need to have the same number of vector components.
+   * A namespace that contains functions that help build more
```

Comment:
(in the sense that they don't build per-se, but help doing it by returning some related data, like `FiniteElementData`)